src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -52,6 +53,7 @@ const AttributeValue = connect(function AttributeValue({
   showAs: string | null
 }) {
   if (showAs === 'currency') return formatCurrency(attributeValue, currency)
+  if (showAs === 'datetime') return formatDatetime(attributeValue)
```

Comment:
* [ ] FYI all of these functions look like helpers but are actually components and could be written as such.

```
if (showAs === 'datetime') return <Datetime value={attributeValue} />
```